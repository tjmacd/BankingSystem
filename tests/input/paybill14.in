login
admin
paybill
Account 1
00001
EC
Text
paybill
Account 1
00001
EC
0.001
logout