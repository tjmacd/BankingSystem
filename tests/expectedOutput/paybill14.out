Please select session type (standard or admin):
Logged in as admin
Please enter account holder's name:
Enter the account number:
Enter the payee company:
Enter the amount to pay:
Invalid amount input
Please enter account holder's name:
Enter the account number:
Enter the payee company:
Enter the amount to pay:
Invalid amount input
Logged out